ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Not a problem. Should that be done in this PR, or when it's officially merged into mainline?